collections/projects.go

```diff
@@ -8,20 +8,26 @@ import (
 
 	"github.com/textileio/go-threads/api/client"
 	s "github.com/textileio/go-threads/store"
+	"github.com/textileio/textile/dns"
 )
 
+var defaultHash = "QmSNWjbDkafwxWUGWTyP1ko4J1CW2WeNfeoiCuTUW2YnDY"
+
 type Project struct {
 	ID              string
 	Name            string
 	Scope           string // user or team
 	StoreID         string
-	Created         int64
+	Domain          string
+	Records         []*dns.Record
```

Comment:
made this an array for max flexibility. DNSlinks for example require 2 records (one for the cname and one for the txt record)